services/movie_session.py

```diff
@@ -0,0 +1,60 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
+        movie=Movie.objects.get(id=movie_id)

```

Comment:
you don't need to perform these queries. You can set cinema_hall and movie by ids and not only objects

Reply:
fixed, but I still don't understand, why I have to use `movie_session_id` with only one underline... Could you suggest me a topic what I probably missed